synchronizations/Synchronization.md

```diff
@@ -0,0 +1,223 @@
+## Tutorial
+![EMF Diagrams](images/EMFDiagrams.png)
+
+We generate the corresponding Ecore files from the two class diagrams shown above. One for the family model and one for the person model. 
+We then generate the NMF models and classes, configure our .csproj file and load our models into the Program.cs file analogous to 
+[Starting our first project](../models/FirstNmfProject.md)
+
+The first step is to create a file in which we define the synchronization logic of the two models. We name this file `Families2PersonsSynchronization.cs`.
+
+Let's think about what we need for our example. For this, let's look at the class diagram and draw the following conclusions. We need the FamilyRegister, the PersonRegister, and the Synchronization class where the synchronization logic between Family and Person is defined.
+```csharp
+class Program
+    {
+        private FamilyRegister familyRegister;
+        private PersonRegister personRegister;
+
+        private ModelRepository repository = new ModelRepository();
+
+        private Families2PersonsSynchronization synchronization = new Families2PersonsSynchronization();
+
+        static void Main(string[] args)
+        {
+        }
+    }
+```
+
+Next, we need to consider how the initialization should look. In our example, this will take place in the class constructor.
```

Comment:
Should look like. What do you mean with "this"?

Reply:
The initialization

Reply:
ok. should look hört sich aber für mich eigenartig an, ich glaube should look like wäre besser